update.sh

```diff
@@ -216,10 +219,31 @@ jobs:
   build:
     name: ${version} on ${variant}
     runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix: \${{fromJson('{\"arch\":${arch_json}}')}}
```

Comment:
Personally prefer the list style because it's easier to see diffs when items are removed. EX:
```yaml
matrix:
  arch:
    - amd64
    - armv6
    ...
```
Not a blocker, and may be painful to do in bash

Reply:
Haha yeah, the pain is why I did it that way originally but I see your point about it being easier to see diffs.